src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -116,6 +116,7 @@ object EasyStageDataset {
         bagRelativePath = s.bagitDir.toPath.relativize(file.toPath).toString
         mime <- readMimeType(bagRelativePath)
         title <- readTitle(bagRelativePath)
+        fileAccessRights <- getFileAccessRights(bagRelativePath)
```

Comment:
Yes, readFileMetadata is called 3 times, but with different arguments. 
The loadBagXML function is the one that is called 3 times with the same arguments...
I will have a look.